setup.py

```diff
@@ -18,19 +18,10 @@
     name='alf',
     version='0.0.1',
     install_requires=[
-        'atari_py == 0.1.7',
-        'fasteners',
-        'gym == 0.12.5',
-        'matplotlib',
-        'numpy',
-        'opencv-python >= 3.4.1.15',
-        'pathos == 0.2.4',
-        'pillow',
-        'psutil',
-        'pybullet == 2.5.0',
-        'tensorboard == 2.1.0',
-        'torch == 1.4.0',
-        'torchvision == 0.5.0',
+        'atari_py == 0.1.7', 'fasteners', 'gym == 0.12.5', 'matplotlib',
```

Comment:
Can you keep the previous format so that changes are easy to spot?

Reply:
It's auto re-formate by pre-commit hook yapf (0.28.0)

Reply:
passed ci code style checking job

Reply:
It's weird. I also use the save version according to CONTRIBUTING.md. But it doesn't format this way.